docs/pages/distribution/publishing-websites.mdx

```diff
@@ -3,12 +3,56 @@ title: Publish websites
 description: Different ways to publish the Expo web app with third-party services.
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
+import { APIBox } from '~/components/plugins/APIBox';
 import { Step } from '~/ui/components/Step';
 import { Tab, Tabs, TabsGroup } from '~/ui/components/Tabs';
+import { Table, TableHead, Row, HeaderCell } from '~/ui/components/Table';
+import { Terminal } from '~/ui/components/Snippet';
+import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
 
 A web app created using Expo can be served locally for testing out the production behavior. Once the testing phase checks out, you can choose from a variety of third-party services to host it.
 
+## Output targets
+
+Expo Router supports three output targets for web apps. The `web.output` target can be configured in the **app.json** file:
```

Comment:
```suggestion
The [`web.output`](/versions/latest/config/app/#output) target can be configured in the [app config](/workflow/configuration/) to set the export method for the web app:
```

Let's add link to the `web.output` config definition and also let's refer app config instead of `app.json`. It's what we use for consistency. Let's also briefly explain what it does.